lenses-alerts-java/src/main/java/io/lenses/alerting/plugin/javaapi/AlertingService.java

```diff
@@ -6,7 +6,29 @@
 import io.lenses.alerting.plugin.Alert;
 import io.lenses.alerting.plugin.javaapi.util.Try;
 
+import java.util.Map;
+
 public interface AlertingService {
+    /**
```

Comment:
you should also update scalaapi `AlertingService`, to make underlying JAlertingService new fields available (they are hidden now)